---
title: Shotgun Metagenomic Workflow landing page
type: 
contributors: 
description: How-to guide for Shotgun Metagenomics workflows on Galaxy Australia.
affiliations: 
---


## About 

This document describes how to use some shotgun metagenomics workflow on galaxy Australia. 

The aim of this workflow is to handle the routine ‘boring’ part of single cell RNAseq data processing. It will produces an ‘AnnData’ object, which can then be used as a base for downstream analysis – either within galaxy or outside of it. AnnData is a standard format used by the ‘scanpy’ python package. 

These workflows represent just one way of processing data for a ‘typical’ scRNAseq experiment – there are many other options!  

The how-to guide is available [here](scrnaseq_wf_guide.html)


## Guide sections

> You can use this section to add navigation tiles for the pages you have created as part of your guide.


## Please cite this guide as follows

> [Citation information goes here]


## Acknowledgements

The workflow is based on the Galaxy Training [Analyses of metagenomics data tutorial](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/general-tutorial/tutorial.html#shotgun-metagenomics-data)

This guide makes use of the ELIXIR toolkit theme: [![theme badge](https://img.shields.io/badge/ELIXIR%20toolkit%20theme-jekyll-blue?color=0d6efd)](https://github.com/ELIXIR-Belgium/elixir-toolkit-theme)


## References

This workflow depends on the following tools and resources:
* MetaPhlAn2: [tutorial](https://github.com/biobakery/biobakery/wiki/metaphlan2), [publication](https://www.nature.com/articles/nmeth.3589)
* HUMAnN2: [tutorial](https://github.com/biobakery/biobakery/wiki/humann2), [publication](https://www.nature.com/articles/s41592-018-0176-y)
* Galaxy: [Galaxy project](https://galaxyproject.org/) 
* Galaxy Australia: [Galaxy Australia service](https://usegalaxy.org.au/)
